_scripts/evaluate_doc_retrieval.py

```diff
@@ -29,6 +30,51 @@ def parse_args() -> argparse.Namespace:
     return parser.parse_args()
 
 
+def ingest_use_cases(eval_path: str) -> pd.DataFrame:
+    """
+    Args:
+        eval_path: path of csv file that contains use cases
+
+    Returns:
+        dataframe that contains use cases to run
+    """
+    # read use cases
+    eval_df = pd.read_csv(args.eval_path, header=0, dtype=str)
+
+    # add expanded expected page number columns
```

Comment:
Just out of curiosity, why do we need to expand the pages?

Reply:
```
""" accepted input: NaN, '3' or '4,8-11' """
```

If we get the string `4,8-11` as input, then we expand to `[4,8,9,10,11]`... then we check if the actual retrieved page was in the list of expected pages (it could have been done differently, but this is how I chose to do it quickly)